data.js

```diff
@@ -0,0 +1,31 @@
+export const data = {
+  "header": {
+    "title-team": "The creative crew",
+    "description-team-title": "Who we are",
+    "description-team-text": "We are team of creatively diverse. driven. innovative individuals working in various locations from the world."
```

Comment:
I know this text is according to the designs, but it makes more sense like this: 
```suggestion
    "description-team-text": "We are a team of creatively diverse-driven innovative individuals working in various locations from the world."
```

Reply:
10e33b8